docs/source/core_concepts/glossary.rst

```diff
@@ -24,6 +24,11 @@ Blocks are organized into a linear sequence over time (also known as the block c
 Blocks are signed with the cryptographic signatures of Iroha `peers <#peer>`__, voting for this block during `consensus <#consensus>`__.
 Signable content is called payload, so the structure of a block looks like this:
 
+.. image:: ../../image_assets/block.png
```

Comment:
I suggest removing the left semi-transparent block from the diagram, since it provides extra explanation for the meaning of previous block hash. 
At the same time, we have very condensed depicture of "block body" (please note that we have not introduced the "body" term. The term is likely is a subject of change).

In my view that introduces a certain level of inconsistency, focusing on non-significant things when some other stuff is omitted at all.

Reply:
Moreover, it is not anyhow depicted that block also stores rejected transactions hashes.

Reply:
P.S.
reason to remove the left part of the diagram: do you feel blind when looking at the diagram, don't you?)

Reply:
P.P.S.
![image](https://user-images.githubusercontent.com/36847043/60599133-96231100-9db6-11e9-9ea1-e56fc3988ec3.png)

Reply:
Fixed :)